### DISCUSSION TITLE
command 'jupyter.execSelectionInteractive' not found

### DISCUSSION THREAD

**Discussion by thomasmikaelsen** [quarto-dev/quarto-cli#6221] [Q&A]
### Description

Hello 

I'm trying to run a cell in VS Code with Python, Jupyter and Quarto. I can render and everything seems to work smoothly, but if I press Run Cell, I get the error _command 'jupyter.execSelectionInteractive' not found_ 

As an example, something as simple as

```{python}
print(2+2)
```

gives the mentioned error.

Thanks a lot for any help!

**Comment by mcanouil** [comment]
With which extension are you running the cell?
What's the version of basically everything are you using?
Are you using a Quarto document or a Jupyter Notebook?
Help us help you by providing information to reproduce or to understand your setup.

Your issue does not seem to have anything to do with Quarto.
This command is a command from the Jupyter extension for VSCode.

PS: I have no issue in both in Jupyter Notebook or a Quarto document.

**Comment by thomasmikaelsen** [comment]
Thank you!

It's a .qmd file and I'm basically just using the code from the [VS Code quarto tutorial](https://quarto.org/docs/get-started/hello/vscode.html) 

I'm using VSC 1.80.1, Python 3.11.4, Quarto 1.3.433

**Reply by mcanouil** [reply]
Could you run `quarto check` and provide the output?

**Reply by thomasmikaelsen** [reply]
Certainly. Indeed, it seems to point to the wrong Python version -- 3.8.17 instead of 3.11.4 which is the version I'm using in my VSC virtual environment. I haven't been able to figure out how to change this

<img width="575" alt="Skærmbillede 2023-07-16 kl  21 41 29" src="https://github.com/quarto-dev/quarto-cli/assets/36076706/dbd05e72-c7fd-4a6e-90d1-9e82a5d37a94">

**Reply by cscheid** [reply]
You can try to diagnose this by opening your VSC shell, printing the environment there (`env` from your favorite unix shell), and comparing it to your out-of-VSC shell.
